#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("background_change_threshold", double_t, 0, "Threshold value for required change", 150, 0, 255)
gen.add("background_learning_rate", double_t, 0, "The learning rate to update the background model", 0.0, 0.0, 0.99)
gen.add("timeout_time", int_t, 0, "The amount of time in seconds that the system is allowed to run for", 30, 0, 300)
gen.add("is_timeout_mode", bool_t, 0, "Run in time out mode", True)
gen.add("is_debug_mode", bool_t, 0, "Run in debugging mode", True)

exit( gen.generate("mcr_background_change_detection", "mcr_background_change_detection_node", "BackgroundChange" ) )